server/src/immich/api-v1/asset/dto/create-asset.dto.ts

```diff
@@ -3,7 +3,10 @@ import { ApiProperty } from '@nestjs/swagger';
 import { Transform, Type } from 'class-transformer';
 import { IsBoolean, IsDate, IsNotEmpty, IsString } from 'class-validator';
 
-export class CreateAssetBase {
+export class CreateAssetDto {
+  @ValidateUUID({ optional: true })
+  libraryId?: string;
+
```

Comment:
Could you sort them alphabetically? pls ;)

Reply:
I want the boolean properties next to each other since that is why I think the decorators got missed.

Reply:
Oh I see. Got it